src/components/TodoFilter.tsx

```diff
@@ -0,0 +1,46 @@
+import cn from 'classnames';
+import { FilterType } from '../types/FilterType';
+
+type Props = {
+  status: FilterType,
+  onStatusChange: (filter: FilterType) => void,
+};
+
+export const TodoFilter: React.FC<Props> = ({ status, onStatusChange }) => {
+  const handleStatusChange
+  = (filterType: FilterType) => () => onStatusChange(filterType);
+
+  return (
+    <nav className="filter">
```

Comment:
consider using map here for rendering links as list in order to avoid code duplication

Reply:
consider the same
<img width="734" alt="image" src="https://github.com/mate-academy/react_todo-app-with-api/assets/70220667/3c4391ca-9193-4eba-828f-b13f91651268">